e2e/scenario/base_vhd_catalog.json

```diff
@@ -12,7 +12,7 @@
         },
         "gen2containerd": {
             "artifactName": "2204-gen2-containerd",
-            "resourceId": "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/2204Gen2/versions/1.1712787789.1362"
+            "resourceId": "/subscriptions/8ecadfc9-d1a3-4ea4-b844-0d9f87e4d7c8/resourceGroups/aksvhdtestbuildrg/providers/Microsoft.Compute/galleries/PackerSigGalleryEastUS/images/2204Gen2/versions/1.1713215700.15013"
```

Comment:
to confirm, this is a 2204gen2 image version?

makes me think, should we also add a new 2404 abe2e scenario as well?

Reply:
change from elsewhere, reverting. will add 2404 e2e once its completely ready with blobfuse support.

Reply:
let's make sure to add a work item for that